source/CMakeLists.txt

```diff
@@ -111,7 +112,7 @@ foreach(build ${DEAL_II_BUILD_TYPES})
 
   #
   # Add compile flags, definitions and (public) feature (recorded in
-  # DEAL_II_TARGETS(|_DEBUG|_RELEASE)).
+  # DEAL_II_TARGETS_(DEBUG|RELEASE)).
```

Comment:
A side note: `DEAL_II_TARGETS(|_DEBUG|_RELEASE)` was not a typo. :smile_cat:

Reply:
Oh, do you want me to revert this change?

Reply:
If you don't mind, yes.